lib/eco_service_web/live/eco_service_live/community_list.ex

```diff
@@ -4,45 +4,73 @@ defmodule EcoServiceWeb.EcoServiceLive.CommunityList do
   alias EcoService.EcoServiceContext
 
   def mount(_params, _session, socket) do
-    params = %{limit: 10, offset: 0}
-    all_communities = EcoServiceContext.fetch_all_communities(params)
+    pag_params = %{limit: 10, offset: 0}
```

Comment:
Please watch this short 1 min video about [magic numbers](https://www.youtube.com/watch?v=jWTJv2Xvo4g)

In Elixir projects, the magic numbers are often set as [module attributes](https://hexdocs.pm/elixir/main/module-attributes.html#as-constants). Now, can you please think about the names of the module attributes that need to be used here?

Reply:
For the limit, I think "records" would be best. Because I am only listing 10 records. 

For offset, "index_of_starting_record".

Reply:
What about just `limit` and `per_page`? How does it ring?

Reply:
Short and simple